src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(int radius) {
+        this.radius = radius;
+    }
+
+    public Circle(int radius, String color) {
+        this.radius = radius;
+        this.color = color;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public String getInfo() {
+        return "Figure: "
+                + type + ", "
+                + "area: "
+                + this.getArea()
+                + " sq. units, "
+                + "radius: "
+                + radius
+                + " units, "
+                + "color: "
+                + color;
+    }
```

Comment:
It seems like the Circle class is missing a 'type' field that is being used in the 'getInfo' method. You should define a 'type' field in this class or in the Figure class if it's common for all figures.